c2corg_api/views/document.py

```diff
@@ -433,6 +433,12 @@ def get(self):
         #    # validate permission (authenticated + associated)
         #    # return 403 if not correct
 
+        title = DBSession.query(DocumentLocale.title) \
```

Comment:
Good catch! The view should indeed raise an error if the requested language is not available.